src/useradd.c

```diff
@@ -1424,7 +1424,7 @@ static void process_flags (int argc, char **argv)
 				/* terminate name, point to value */
 				*cp = '\0';
 				cp++;
-				if (putdef_str (optarg, cp) < 0) {
+				if (putdef_str (optarg, cp, NULL) < 0) {
```

Comment:
`printf("%s", NULL);` is Undefined Behavior.  Printing `(null)` is a GNU extension, which may not work if the call is optimized to puts(3) or something else.

Reply:
<https://stackoverflow.com/a/11589479/6872717>